measures/ApplyUpgrade/measure.rb

```diff
@@ -121,7 +127,7 @@ def run(model, runner, user_arguments)
     end
 
     upgrade_name = runner.getStringArgumentValue('upgrade_name', user_arguments)
-
+    args = runner.getArgumentValues(arguments(model), user_arguments)
```

Comment:
If you're going to access new `:project_name` from the `args` hash, I'd suggest using `args[:upgrade_name]` -- therefore you can delete the `upgrade_name = ...` line.